README.md

```diff
@@ -1,2 +1,5 @@
 # Project_FE
-## This is my first project!
\ No newline at end of file
+## This is my first project!
+*** hello world ***
+ergeg
```

Comment:
THis is redundant text. Can you remove it?

Reply:
zdelal